collection-scripts/gather_db

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# When called from the shell directly
+if [[ -z "$DIR_NAME" ]]; then
+    CALLED=1
+    DIR_NAME=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+    source "${DIR_NAME}/common.sh"
+fi
+
+# Default DB DUMP Option
+DB_OPT="--single-transaction --complete-insert --skip-lock-tables --lock-tables=0"
+DB_DUMP=${BASE_COLLECTION_PATH}/dbs
+
+function dump_db {
+    local dbpod="$1"
+    local ns="$2"
+    local dbpass="$3"
+    local dbname="$4"
+    [[ -z $dbname ]] && dump="openstack_databases" || dump=$dbname
```

Comment:
-1: I think it's easier to read as:

```
local dump="${4:-openstack_databases}"
```

nit: Maybe instead of `dump` use `filename` or `dump_filename`.

Reply:
Done (and thanks for the hint here!)